cocos/renderer/core/gfx/GFXCommandBuffer.h

```diff
@@ -32,7 +32,8 @@ class CC_CORE_API GFXCommandBuffer : public GFXObject {
   virtual void updateBuffer(GFXBuffer* buff, void* data, uint size, uint offset = 0) = 0;
   virtual void copyBufferToTexture(GFXBuffer* src, GFXTexture* dst, GFXTextureLayout layout, const GFXBufferTextureCopyList& regions) = 0;
   virtual void execute(const std::vector<GFXCommandBuffer*>& cmd_buffs, uint32_t count) = 0;
-  
+  virtual void enableDepthBias(bool enabled) { _depthBiasEnabled = enabled; }
```

Comment:
I think use setDepthBias is better.

Reply:
there is already a `setDepthBias ` to set depth bias